### ISSUE TITLE
TypeError: no loader for this environment specified 

### ISSUE THREAD

**Issue by 0xmihutao** [character-ai/prompt-poet#18]
I get this error when I try to initialize a `Prompt` from a raw template that contains `{% include '...' %}`, but I don't get this when I have the exact same contents in a file and I try to initialize a `Prompt` object via file path instead.

**Comment by nitinimage** [comment]
Use 'template_path' parameter instead of 'raw_template'.

**Comment by 0xmihutao** [comment]
yes I know, but my complaint is that it shouldn't be the case that I have to load it from a file. My use case is in passing a template through an API endpoint, but now I have to write it to a temporary file and pass it in via `template_path` which is not desirable
